airflow/www/static/js/dag_dependencies.js

```diff
@@ -37,6 +37,12 @@ const g = new dagreD3.graphlib.Graph()
   })
   .setDefaultEdgeLabel(() => ({ lineInterpolate: 'basis' }));
 
+// Sometimes the data doesn't load when the scheduler isn't running.
+if (!nodes.length) {
+  $('#error').show();
+  $('#error_msg').text('There was an issue loading dependencies. Make sure everything is up and running, then try again.');
```

Comment:
Does this message show up if everything is running but there are just no dags?

(I.e. if Load_examples is false but scheduler is up)

Reply:
I'll update the copy to say make sure you have dags.